appcircle_ios_reactnative-codepush/1.1.0/component.yaml

```diff
@@ -0,0 +1,77 @@
+platform: common
+buildPlatform: ReactNative
+displayName: Appcircle CodePush
+description: "Release a React Native update to Appcircle CodePush."
+docsUrl: https://docs.appcircle.io
```

Comment:
Before this feature comes out, we need to prepare the document and add the link.